pos_product_template/models/pos_config.py

```diff
@@ -14,3 +14,9 @@ class PosConfig(models.Model):
         help="If selected the product variant selection screen will show the variants,"
         " else it will only allow to confirm once all the attributes are chosen.",
     )
+
+    iface_show_product_template = fields.Boolean(
+        string="Shows product template in TPV",
```

Comment:
TPV is not english.
as it is in pos_config, I guess "Show Product Templates" is enough.